### PR TITLE
Filter for a user assigned tickets

### PR DESCRIPTION
**Pull request by FredyLy**

### CODE REVIEW

File: src/components/projects/ProjectList.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import CustomFilterCheckBox from '../../assets/custom-components/CustomFilterCheckBox';
+
+function ProjectList(): JSX.Element {
+  return (
+    <div style={{ margin: '2rem 3rem 0 3rem' }}>
+      <TitleBar title="Projects" onClickRigthBtn={() => console.log('click click')} />
+      <div>
+        <CustomFilterCheckBox
+          label="Assigned to me"
+          onClick={() => console.log('click')}
```
Comment (grubux) [inline src/components/projects/ProjectList.tsx:14]: To remove

Comment (grubux) [reply src/components/projects/ProjectList.tsx:14]: le console.log()

File: src/components/tasks/AddTaskCard.tsx [hunk 2]
```diff
@@ -57,7 +56,9 @@ function AddTaskCard({ toggleDisplay }: IAddTaskCard): JSX.Element {
   });
   const [pickDeadline, setPickDeadline] = useState<Date | null>(new Date());
   const [selectStatus, setSelectStatus] = useState<string>('');
-  const [selectProject, setSelectProject] = useState<GetTicketsProjects_getAllProjects | null | undefined>(null);
+  const [selectProject, setSelectProject] = useState<
+    GetTicketsProjects_getAllProjects | null | undefined
```
Comment (grubux) [inline src/components/tasks/AddTaskCard.tsx:60]: undefined ?

Comment (FredyLy) [reply src/components/tasks/AddTaskCard.tsx:60]: Code de Sophie. On peut le changer par autre chose.

File: src/components/projects/ProjectList.tsx (hunk 1 above)
Comment (FredyLy) [reply src/components/projects/ProjectList.tsx:14]: This page has not been developed yet. I'll let it for the moment. It has to be replace with the right function to call.

**Review by grubux — APPROVED** [review]

**Pull request merged**